appinfo/routes.php

```diff
@@ -22,7 +22,7 @@
 return [
 	'routes' => [
 		['name' => 'site#showDefaultPage', 'url' => '/', 'verb' => 'GET'],
-		['name' => 'site#showPage', 'url' => '/{id}', 'verb' => 'GET'],
```

Comment:
This route needs to be kept as all path parameters are mandatory. Without it just passing no path would no longer work

Reply:
Seems like you are right. I would have expected the default to be any number of characters, but apparently it needs at least one to match if no requirement is given.